src/pages/bounties/[id]/[[...slug]].tsx

```diff
@@ -633,6 +627,46 @@ const useStyles = createStyles((theme) => ({
   },
 }));
 
+const BountyEntries = ({ bounty }: { bounty: BountyGetById }) => {
+  const theme = useMantineTheme();
+  const entryCreateUrl = `/bounties/${bounty.id}/entries/create`;
+
+  return (
+    <Stack spacing="xl" py={32}>
+      <Group position="apart">
+        <Title order={2} size={28} weight={600}>
+          Hunters
+        </Title>
+        <Button size="xs">Submit</Button>
+      </Group>
+      <Paper
+        p="xl"
+        radius="sm"
+        sx={{
+          display: 'flex',
+          alignItems: 'center',
+          justifyContent: 'center',
+          background: theme.colorScheme === 'dark' ? theme.colors.dark[5] : theme.colors.gray[1],
+        }}
+        component={NextLink}
+        href={entryCreateUrl}
```

Comment:
Since we already have link in the Button, this would actually throw a browser warning/error because you can't have links inside links. Also, it'll be a pain to handle for muted users 😛 I would just drop it